.github/workflows/macos.yml

```diff
@@ -11,9 +11,24 @@ on:
       - '**'
 
 jobs:
+  check_skip_ci:
+    name: 'Check for the "skip ci" pattern'
+    runs-on: ubuntu-latest
+    outputs:
+      run_tests: ${{ steps.check.outputs.run_tests }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check for the "skip ci" pattern
+        id: check
+        run: |
+          commits=${{ join(github.event.commits.*.message, ', ') }}
```

Comment:
This will inhibit the test to be run if *any* of the commit-messages contains the marker. Thus if you add further commits to a pull-request, die tests will still not be triggered. Travis only considers the latest commit message AFAIK.

Reply:
Yes, agreed. However I'm already working on another (reusable) solution for this, I'll get back to you soon. @BoboTiG hold off on this for a couple of days.